bfgg-site/src/components/PastTestsTable/index.jsx

```diff
@@ -0,0 +1,214 @@
+              <TableCell
+                  key={headCell.id}
+                  align='center'
+                  padding='default'
+                  sortDirection={orderBy === headCell.id ? order : false}
+              >
+                <TableSortLabel
+                    active={orderBy === headCell.id}
+                    direction={order}
+                    onClick={createSortHandler(headCell.id)}
+                >
+                  {headCell.label}
+                  {orderBy === headCell.id ? (
+                      <span className={classes.visuallyHidden}>
+                  {order === 'desc' ? 'sorted descending' : 'sorted ascending'}
+                </span>
+                  ) : null}
+                </TableSortLabel>
+              </TableCell>
+          ))}
+        </TableRow>
+      </TableHead>
+  );
+}
+
+EnhancedTableHead.propTypes = {
+  classes: PropTypes.object.isRequired,
+  onRequestSort: PropTypes.func.isRequired,
+  order: PropTypes.oneOf(['asc', 'desc']).isRequired,
+  orderBy: PropTypes.string.isRequired,
+};
+
+const useToolbarStyles = makeStyles(theme => ({
+  root: {
+    paddingLeft: theme.spacing(2),
+    paddingRight: theme.spacing(1),
+    marginTop: theme.spacing(2),
+  },
+  highlight:
+      theme.palette.type === 'light'
+          ? {
+            color: theme.palette.secondary.main,
+            backgroundColor: lighten(theme.palette.secondary.light, 0.85),
+          }
+          : {
+            color: theme.palette.text.primary,
+            backgroundColor: theme.palette.secondary.dark,
+          },
+  title: {
+    flex: '1 1 100%',
+  },
+}));
+
+const EnhancedTableToolbar = (props) => {
+  const classes = useToolbarStyles();
+  const { getPastTests } = props;
+
+  return (
+      <Toolbar
+          className={clsx(classes.root)}
+      >
+        <Typography className={classes.title} variant="h6" id="testsTableTitle">
+          Past Tests
+        </Typography>
+        <IconButton onClick={getPastTests}>
+          <RefreshIcon/>
+        </IconButton>
+      </Toolbar>
+  );
+};
+
+EnhancedTableToolbar.propTypes = {
+  getPastTests: PropTypes.func.isRequired,
+};
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    width: '100%',
+  },
+  paper: {
+    width: '100%',
+    marginBottom: theme.spacing(2),
+  },
+  table: {
+    minWidth: 750,
+  },
+  tableWrapper: {
+    overflowX: 'auto',
+  },
+  visuallyHidden: {
+    border: 0,
+    clip: 'rect(0 0 0 0)',
+    height: 1,
+    margin: -1,
+    overflow: 'hidden',
+    padding: 0,
+    position: 'absolute',
+    top: 20,
+    width: 1,
+  },
+}));
+
+export default function PastTestsTable(props) {
+  const {selectedGroup, setSnackbar, setSnackbarOpen } = props;
+  const classes = useStyles();
+  const [order, setOrder] = React.useState('asc');
+  const [orderBy, setOrderBy] = React.useState('StartTime');
+  const [ tests, setTests ] = useState([]);
+
+  const handleRequestSort = (event, property) => {
```

Comment:
seems like this function doesnt work properly, the sorting isnt updating when clicking different column headers

Reply:
😭